quic/s2n-quic-core/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "s2n-quic-core"
-version = "0.18.0"
+version = "0.18.1"
```

Comment:
I like the idea of trying to stick to semver for all of the crates, because I think that's easier to reason about. 

E.g. "we follow semver for all of the crates that we publish" rather than "we follow semver for the primary crate that we publish and but not for published, non-primary crates"

Reply:
Although I admit that figuring out whether things should be minor/patch bumps is currently rather tedious, the solution I would push for is automating version bumps through the conventional commit parsing.